providers/bigbasket/bigbasket.go

```diff
@@ -0,0 +1,25 @@
+package github
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("bigbasket", &Provider{})
+}
+
+// Provider merely implements the Provider interface.
+type Provider struct{}
+
+// BuildURI generates a search URL for Bigbasket.
+func (p *Provider) BuildURI(q string) string {
+	return fmt.Sprintf("https://www.bigbasket.com/ps/?q=%s", url.QueryEscape(q))
+}
+
+// Tags returns the tags relevant to this provider.
+func (p *Provider) Tags() []string {
+	return []string{"shopping"}
```

Comment:
Right now shopping is mainly for general purpose goods. For now I would keep it with no tags. If we add more grocery sites we can add a new tag then. =)

Reply:
Removed "shopping" and squashed my commits into a single one.